packages/angular/src/generators/init-mfe/lib/generate-config.ts

```diff
@@ -0,0 +1,26 @@
+import type { Tree } from '@nrwl/devkit';
+import type { Schema } from '../schema';
+import { generateFiles, joinPathFragments } from '@nrwl/devkit';
+
+const SHARED_SINGLETON_LIBRARIES = [
+  '@angular/core',
+  '@angular/common',
+  '@angular/common/http',
+  '@angular/router',
+];
+
+export function generateWebpackConfig(
+  host: Tree,
+  options: Schema,
+  appRoot: string,
+  remotesWithPorts: { remoteName: string; port: number }[]
+) {
+  generateFiles(host, joinPathFragments(__dirname, '../files'), appRoot, {
```

Comment:
Similar to what I mentioned in another comment, if a previous custom Webpack configuration was present, this will overwrite it.

Reply:
Only if it's at the same path. I think this is fine. 

We don't make a `git commit` so if someone mistakenly uses this on the wrong app they can just discard the changes.

The onus is on the user. They will likely know what this generator does otherwise they wouldn't be using it, so they'll know that it's going to generate a webpack.config.js

Reply:
I agree there are things the dev would have to do, if someone is going the route of using MFEs, they definitely need to know what's going on, but I do think is not a great experience to just overwrite things and not even tell the dev what happened or why did it happen.

I think it doesn't hurt to log a warning, it would actually be a better experience, it's immediate feedback on what happened, devs wouldn't have to realize by themselves what happened and maybe even think is a bug. A warning will communicate that's by design (not a bug) and will immediately bring their attention to it.

Reply:
Yes, we should probably log a warning about it :)